ts/util/cleanSearchTerm.ts

```diff
@@ -4,7 +4,7 @@
 export function cleanSearchTerm(searchTerm: string): string {
   const lowercase = searchTerm.toLowerCase();
   const withoutSpecialCharacters = lowercase.replace(
-    /([!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~])/g,
+    /([-!"#$%&'()*+,./\\:;<=>?@[\]^_`{|}~])/g,
```

Comment:
It looks like some of the characters here were reordered. Why is that?

Reply:
Hey, yeah I just noticed that in this grouping the `-` character specifies a range technically unless it's the first or last character in the brackets, so I just moved it up front. I think it probably doesn't matter in this case, but it seemed better to explicitly specify it as a character literal.

I'm referencing [this documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Cheatsheet) in the groups and range section: 
> You can specify a range of characters by using a hyphen, but if the hyphen appears as the first or last character enclosed in the square brackets it is taken as a literal hyphen to be included in the character class as a normal character.

Reply:
I could be wrong, but the range `,-.` seems to be the characters `,`, `-` and `.`, since `-` is between `,` and `.` in the ASCII table (please do test it if you rely on this indication).

Reply:
That's true I didn't think about that, but wouldn't it be equivalent then? So in the end the 3 characters `,`, `-` and `.` be replaced by a whitespace. I can change the ordering back also

Reply:
> That's true I didn't think about that, but wouldn't it be equivalent then? So in the end the 3 characters `,`, `-` and `.` be replaced by a whitespace. I can change the ordering back also

Yes that's what I meant, both your changes and the original code should be equivalent for these 3 characters (again, assuming that I'm correct).

Reply:
Makes sense. Thanks for explaining.